src/mrgw.sh

```diff
@@ -21,7 +22,7 @@ shift 2
 
 run_root=$script_root/run/$name
 pidfile=$run_root/out/radosgw.${port}.pid
-asokfile=$run_root/out/radosgw.${port}.asok
+asokfile=$($ceph_bin/ceph-conf --show-config-value admin_socket --name radosgw.${port})
```

Comment:
@tchaikov why is it a problem for these .asok files to go under `$run_root` with the others? this script doesn't run as part of make check

this change causes `ceph-conf` to fail with:
> error parsing 'radosgw.8000': expected string of the form TYPE.ID, valid types are: auth, mon, osd, mds, mgr, client